bugsnag-android-core/src/main/java/com/bugsnag/android/Stackframe.kt

```diff
@@ -5,43 +5,86 @@ import java.io.IOException
 /**
  * Represents a single stackframe from a [Throwable]
  */
-class Stackframe @JvmOverloads internal constructor(
-
+class Stackframe : JsonStream.Streamable {
     /**
      * The name of the method that was being executed
      */
-    var method: String?,
+    var method: String?
+    set(value) {
+        field = value
+        val frame = this.nativeFrame
```

Comment:
Kotlin has a safe call operator which performs this null check behind the scenes. Therefore this could be stripped down a bit to: `nativeFrame?.file = value`